docs/creating-service-bindings/creating-service-binding.md

```diff
@@ -5,7 +5,115 @@ sidebar_position: 1
 # Creating Service Binding
 
 In this section, we consider that the service you want to connect to is already exposing binding metadata by either:
-- Beeing part of the "known" services
+- Being part of the "known" services
 - Exposing binding-metadatas by default
 If it's not the case, please refer to the following section(#)
 
+
+### Service Binding Resource
+
+```yaml
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+
+metadata:
+  name: binding-request 
+
+spec:
+
+  application:
+    name: nodejs-app
+    group: apps
+    version: v1
+    resource: deployments
+
+  services:
+  - group: postgresql.dev
+    version: v1alpha1
+    kind: Database
+    name: db-demo
+```
+
+There are 2 parts in the request:
+- application
+- services
+
+### Application
+
+An application can be referenced either by name or by labels.
+
+#### Reference by Name
+
+```
+  application:
+    name: nodejs-app
+    group: apps
+    version: v1
+    resource: deployments
+```
+
+#### Reference by Label Selector
+
+```
+  application:
+    matchLabels:
```

Comment:
Can I have multiple matchLabels? 
How should that be formatted?

Reply:
This is a `matchLabel` on the application. SBO doesn't have anything to do with this.

Reply:
SBO doesn't support selecting applications through labels.